keras_retinanet/preprocessing/generator.py

```diff
@@ -112,19 +116,32 @@ def resize_image(self, image):
     def preprocess_image(self, image):
         return preprocess_image(image)
 
-    def preprocess_group(self, image_group, annotations_group):
-        for index, (image, annotations) in enumerate(zip(image_group, annotations_group)):
-            # preprocess the image (subtract imagenet mean)
-            image = self.preprocess_image(image)
+    def preprocess_group_entry(self, image, annotations):
+        # preprocess the image
+        image = self.preprocess_image(image)
+
+        # randomly transform both image and annotations
+        if self.transform_generator is not None:
```

Comment:
Why not use `if self.transform_generator:` ?

Reply:
No reason really. Do you prefer it like that?

Reply:
Yeah.

Reply:
Updated.